config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: {omniauth_callbacks: 'users/omniauth_callbacks'}
   root to: 'home_page#show'
+  resources :transactions
```

Comment:
@sinpai do you need all resources or just few of them? [check notes](http://guides.rubyonrails.org/routing.html#restricting-the-routes-created)

Reply:
Yes, I will shorten variants with only: []. 👍